STTR versioning

afjlambert opened the pull request:
Waiting for FL to go into production.
Run `STTR_ENV=production npm run sttr` before merging this PR.

afjlambert committed:
Implement sttr versioning

robinpiets commented on apps/client/src/sttr_client/models/permit.js:
```diff
@@ -1,3 +1,5 @@
+import { isNumber } from "util";
```
Why use `isNumber` from `util` and not from `lodash.isnumber`, like you do in `models/question.js`? Is this different or maybe more lightweight?

afjlambert replied:
Good point! My IDE imported this function from `util` which I didn't even know existed. Anyway its been deprecated since node v4.0.0. I'll replace it with lodash's function.

robinpiets commented on apps/client/sttr_build/index.js:
```diff
@@ -18,7 +18,7 @@ const outputDir = path.join(
 const sttrApi = `https://sttr-builder${
   env === "PROD" ? "" : "-staging"
-}.eu.meteorapp.com/api`;
+}.eu.meteorapp.com/api/v2`;
```
v2 is already on production?

robinpiets replied:
Never mind

robinpiets commented on apps/client/sttr_build/index.js:
```diff
@@ -77,31 +77,37 @@ function checkStatus(res) {
+      permits: permits.map((permit) => {
+        const res = permitsConfig.find((p) => p.id === permit.id);
+        return {
+          version: res.json.version,
```
Could there be a case where `res.json` or `res.json.version` is `undefined`?

afjlambert replied:
No there shouldn't be a reason for it to be undefined. If it does however the script should exit properly. Will double-check if that's the case. Thanks.

robinpiets commented on apps/client/sttr_build/index.js:
```diff
@@ -67,7 +67,7 @@ function checkStatus(res) {
       return json.map(({ _id }) => _id);
```
Is there any advantage to also write the version in the `topics.source.json` file?

afjlambert committed:
Implement PR-feedback

- Replace util.isnumber with lodash version
- Better json input validation

dependabot[bot] committed:
Chore(deps): Bump lodash from 4.17.15 to 4.17.19 (#305)

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.19.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.19)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Chore(deps): Bump lodash from 4.17.15 to 4.17.19 in /apps/graphql (#304)

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.19.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.19)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

afjlambert committed:
Merge remote-tracking branch 'origin/develop' into feature/sttr-versioning

robinpiets commented:
Please Squash and Merge

afjlambert committed:
Merge branch 'chappie2.0' into feature/sttr-versioning

afjlambert committed:
Merge remote-tracking branch 'origin/chappie2.0' into feature/sttr-versioning

afjlambert committed:
Add version numbers to checkers

afjlambert committed:
Merge remote-tracking branch 'origin/chappie2.0' into feature/sttr-versioning
